Client/Frontend/Browser/MainMenuActionHelper.swift

```diff
@@ -356,34 +358,30 @@ class MainMenuActionHelper: PhotonActionSheetProtocol, FeatureFlaggable, CanRemo
     }
 
     private func getSendToDevice() -> PhotonRowActions {
+        typealias sendDelegate = InstructionsViewDelegate & DevicePickerViewControllerDelegate
         return SingleActionViewModel(title: .AppMenu.TouchActions.SendLinkToDeviceTitle,
                                      iconString: ImageIdentifiers.sendToDevice) { _ in
-            guard let bvc = self.menuActionDelegate as? InstructionsViewDelegate & DevicePickerViewControllerDelegate
+            guard let bvc = self.menuActionDelegate as? sendDelegate,
```

Comment:
Same comment for delegate. This assumes the menuActionDelegate is something else. The day we decouple bvc this will make this more complicated to manage as this would break behavior without warning us (would just silently fall into the else case). I think here we should create a new delegate to handle this case rather than reuse the menuActionDelegate which is already managing a lot.